connect/src/main/java/com/stripe/android/connect/analytics/ConnectAnalyticsService.kt

```diff
@@ -0,0 +1,60 @@
+package com.stripe.android.connect.analytics
+
+import android.app.Application
+import com.stripe.android.core.BuildConfig
+import com.stripe.android.core.Logger
+import com.stripe.android.core.networking.AnalyticsRequestV2
+import com.stripe.android.core.networking.AnalyticsRequestV2Factory
+import com.stripe.android.core.networking.DefaultAnalyticsRequestV2Executor
+import com.stripe.android.core.networking.DefaultStripeNetworkClient
+import com.stripe.android.core.networking.RealAnalyticsRequestV2Storage
+import com.stripe.android.core.utils.RealIsWorkManagerAvailable
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+
+/**
+ * Service for logging [AnalyticsRequestV2] for the Connect SDK.
+ * This service is very simple. Consumers should prefer [ComponentAnalyticsService] instead,
+ * which uses this service internally.
```

Comment:
This docstring is inaccurate.

```suggestion
 * Analytics service configured for Connect SDK.
```

Reply:
What's inaccurate about this docstring? Is it specifically mentioning AnalyticsRequestV2, since that's the type we log but not the type the service exposes?

I think it's still useful to recommend consumers to ComponentAnalyticsService (which is what should be used in other parts of the SDK anyways), let me know what you think of the updated docstring.

Reply:
Oh, I might've misread it. I thought it was saying `ConnectAnalyticsService` uses `ComponentAnalyticsService` internally, but it's saying the opposite (which is true).

Still, I'm not a fan of referencing downstream dependencies since it's not clean and adds maintenance cost.